### ISSUE TITLE
Long non-breaking post title does not look good on WordPress.com Reader view

### ISSUE THREAD

**Issue by arunsathiya** [Automattic/wp-calypso#35834]
For a post with a long, non-breaking title like `Super/Duper/Long/NonBreaking/Path/Name/To/A/File/That/Is/Way/Deep/Down/In/Some/Mysterious/Remote/Desolate/Part/Of/The/Operating/System/To/A/File/That/Just/So/Happens/To/Be/Strangely/Named/.txt`, the title overflows the available screen space to the right.

This can be seen on WordPress.com Reader view similar to the pages view - https://github.com/Automattic/wp-calypso/issues/35828

<img width="1252" alt="Screenshot 2019-08-28 at 16 43 42" src="https://user-images.githubusercontent.com/18581859/63851123-3d858400-c9b3-11e9-97ba-ffc9ca0baf9b.png">

**Comment by bluefuton** [comment]
Good catch @arunsathiya! Adding in an `overflow-wrap: break-word;` should do the trick I think.

**Comment by anuragbhd** [comment]
Thanks for the report, @arunsathiya. I have created a PR that fixes this and the related issue #35828.

@bluefuton `overflow-wrap: break-word;` indeed fixes the issue on Reader page, but on /pages only `word-break: break-word;` works.

**Comment by arunsathiya** [comment]
I was meaning to get back to this, but got sidetracked with other pressing works. Thanks for the patches, @anuragbhd!
